gprofiler/profilers/java.py

```diff
@@ -69,6 +77,18 @@ def get_ap_version() -> str:
     return Path(resource_path("java/async-profiler-version")).read_text()
 
 
+def _locate_hotspot_error_file(process: Process) -> str:
+    error_file = "hs_err_pid%p.log"
+    for arg in process.cmdline():
+        if arg.startswith("-XX:ErrorFile"):
+            _, error_file = arg.split("=", maxsplit=1)
```

Comment:
Can it also be `-XX:ErrorFile xxx`? Or only `=`

Reply:
```
$ java -XX:ErrorFile /tmp/test
Improperly specified VM option 'ErrorFile'
```